app/javascript/plugins/fundraiser/FundraiserView.js

```diff
@@ -45,6 +48,17 @@ export class FundraiserView extends Component {
     }
   }
 
+  supportedLocalCurrency() {
+    const country = window.champaign.personalization.member.country;
```

Comment:
Can we be sure that member will be present at this point? is it worth adding a null check? `window.champaign.personalization.member?.country || some_default`

Reply:
I started implementing this and in the middle realized that it doesn't really make sense. We can offer the currency disclaimer if we know what the local currency is and can distinguish whether we support it or not. If we don't know the country, we don't know what the local currency is. If window.champaign.personalization.member is not there, then window.champaign.personalization.member.country will be undefined - in the next row we have an early return in case we know that we don't support the local currency -> `if (!Object.keys(localCurrencies).includes(country)) return true` - this will just return true if the member's country is not known (it's the best we can do).